CHANGELOG.md

```diff
@@ -52,7 +52,7 @@
 ### 🧰 Bug fixes 🧰
 
 - Fixed bug where `telemetryInitializer` is not cleaned up when `newService` errors (#6239)
-
+- (Builder)Remove setting of the Go executable from Validate fn (#6273)
```

Comment:
This is an internal refactoring. Doesn't need a changelog entry

Reply:
Fixing

Reply:
@dmitryax can you please check again?